src/api/workers/enrichment.js

```diff
@@ -51,19 +51,27 @@ const worker = async (id, ctx) => {
 
     return async entry => {
         const enricherTransformer = await transformerFactory(id);
+        const enrichment = await ctx.enrichment.findOneById(id);
+        await ctx.dataset.updateOne(
+            { _id: new ObjectId(entry._id) },
+            { $set: { [enrichment.name]: 'En attente' } },
+        );
 
         try {
-            const { value, enrichment } = await enricherTransformer({
+            let { value } = await enricherTransformer({
                 id: entry._id,
                 value: entry,
             });
 
+            if (value === undefined) {
+                value = 'n/a';
```

Comment:
Are we sure we want n/a in the field? If I remember correctly, front end doesn't display fields that have a null value.
Here it would show an error field instead.

Reply:
Yes, seen with the Lodex team on mattermost yesterday :)
For them if the value is null it means that nothing happened, so they didn't know if the webservice was called